lib/services/notify/slack.ts

```diff
@@ -0,0 +1,14 @@
+import { env } from '@/env.mjs';
+import { ofetch } from 'ofetch';
+
+export const notifyViaSlack = async (message: string) => {
```

Comment:
not sure about the name

Reply:
maybe `notifySlack`? In theory we're notifying slack :D

Reply:
👍